New format of documentation (#369)
husarion/panther_ros: panther_bringup/README.md

| `controller_manager`      | Controller Manager performs two main functions. First, it manages controllers and their required interfaces, handling tasks like loading, activating, deactivating, and unloading. Second, it interacts with hardware components, ensuring access to their interfaces. For more information, refer to  [controller_manager](https://control.ros.org/master/doc/ros2_control/controller_manager/doc/userdoc.html). This node manages the: `imu_broadcaster`, `joint_state_broadcaster`, `drive_controller` <br/> *[controller_manager/controller_manager](https://github.com/ros-controls/ros2_control/blob/master/controller_manager)* |

> | `drive_controller`        | Manages mobile robots with a differential drive. It converts velocity commands for the robot body into wheel commands for the base. It also calculates odometry from hardware feedback and shares it. <br/> *[diff_drive_controller/diff_drive_controller](https://github.com/ros-controls/ros2_controllers/tree/master/diff_drive_controller)*                                                                                                                                                                                                                                                                                        |

Reviewer: This can also be a mecanum drive controller depending on wheel_type. It may be worth to mention that in description.

| `joint_state_broadcaster` | Reads all state interfaces and reports them on specific topics. <br/> *[joint_state_broadcaster/joint_state_broadcaster](https://github.com/ros-controls/ros2_controllers/tree/master/joint_state_broadcaster)*                                                                                                                                                                                                                                                                                                                                                                                                                        |

> | `lights_container`        | Node for dynamically loadable components such as plugins <br/>[*rclcpp_components/component_container*](https://github.com/ros2/rclcpp/tree/rolling/rclcpp_components)                                                                                                                                                                                                                                                                                                                                                                                                                                       |

Reviewer: ```suggestion
| `lights_container`        | Node for managing ROS components. This node menages: `lights_controller`, `lights_driver`. <br/>[*rclcpp_components/component_container*](https://github.com/ros2/rclcpp/tree/rolling/rclcpp_components)                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

# panther_bringup

The package contains the default configuration and launch files necessary to start all the basic functionalities of the Husarion Panther robot.

## Launch Files

This package contains:

> - `bringup.launch.py`: Responsible for activating whole robot system.

Reviewer 2: I'm not sure why we moved the whole bringup package description to master readme file?

Author: I don't know what things you're talking about, but keeping additional issues in bringup in the previous version would cause a lot of redundancy. I will divide the answer into 3 points because previously bringup contained 3 things:

1. Default Nodes Launched
If we wanted to maintain the standard in all packs, the bringup and simulation launch packages would be huge and would contain descriptions of nodes from other packages because they are used.

2. Bringup Launch Arguments
Similarly to the above, bringup and simulation have almost identical arguments and it is best to indicate this in the place where they are launched. There would also be a lot of redundancy because we would have to describe namespace and use_sim args in almost every file.

3. Description of nodes
Currently, bringup does not launch any nodes, so it is not needed.
